### PR TITLE
Malo

### PR DESCRIPTION
**Pull request by malo-renaudin**

### CODE REVIEW

**Comment by victoris93** [comment]
@malo-renaudin 1. could you update `augmentations.py` to the current version in my branch? also, could you make sure each aug function returns a 3D array of shape (1, 1000, 1000)? 2. You dded python cache files, could you remove them? 3. Could you summarize here what changes you mande in `barbano_aug.ipynb`? They are too large to be loaded.

**Comment by demianw** [comment]
@victoris93 , maybe you should add a `.gitignore` file such that unwanted files are not added 

---

**Comment by victoris93** [comment]
@demianw a good idea, but I rarely do `git add *`. So @malo-renaudin if you want you can indeed create a `.gitignore`

**Comment by demianw** [comment]
> @malo-renaudin 1. could you update `augmentations.py` to the current version in my branch? also, could you make sure each aug function returns a 3D array of shape (1, 1000, 1000)? 2. You dded python cache files, could you remove them? 3. Could you summarize here what changes you mande in `barbano_aug.ipynb`? They are too large to be loaded.

@victoris93 maybe you should add a `.gitignore` file to the repo such that we only keep the files we need.

Also, if we agree on the API and the dimensions, maybe we could document them? Numpydoc style? See the examples en https://numpydoc.readthedocs.io/en/latest/example.html

**Comment by demianw** [comment]
> @demianw a good idea, but I rarely do `git add *`. So @malo-renaudin if you want you can indeed create a `.gitignore`

Still, when you work in a team, it's a good habit :)
